### PR TITLE
5 minute cache expire headers between 20:00 and midnight

### PR DESCRIPTION
**Pull request by kyokukou**

### CODE REVIEW

File: browse/services/listing/__init__.py [hunk 1]
```diff
@@ -295,7 +295,10 @@ def gen_expires() -> str:
        RFC 1123 format ex 'Wed, 21 Oct 2015 07:28:00 GMT'
     """
     now=datetime.now().astimezone(ZoneInfo("US/Eastern"))
-    midnight=now.replace(hour=0, minute=0, second=0, microsecond=0)
-    expire = midnight + timedelta(days=1)
+    if now.hour>=20:
+        expire= now+ timedelta(minutes=5)
+    else:
+        midnight=now.replace(hour=0, minute=0, second=0, microsecond=0)
```
Comment (cbf66) [inline browse/services/listing/__init__.py:301]:
```suggestion
        midnight=now.replace(hour=20, minute=0, second=0, microsecond=0)
```

File: browse/services/listing/__init__.py [hunk 2]
```diff
@@ -295,7 +295,10 @@ def gen_expires() -> str:
...
+        expire = midnight + timedelta(days=1)
```
Comment (cbf66) [inline browse/services/listing/__init__.py:302]:
```suggestion
        expire = midnight + timedelta(days=0)
```

Comment (cbf66) [inline browse/services/listing/__init__.py:302]:
```suggestion
        announcetime=now.replace(hour=20, minute=0, second=0, microsecond=0)
        expire = announcetime
```

**Review by cbf66 — CHANGES_REQUESTED** [review]
Prior to 20:00 everything expires at 20:00. Otherwise +5 minutes.

There's no midnight expire anymore.

File: browse/services/listing/__init__.py (hunk 2 above)
Comment (kyokukou) [reply browse/services/listing/__init__.py]: oops- how about now

**Review by cbf66 — APPROVED** [review]

**Pull request merged**